js/application.js

```diff
@@ -1,250 +1,238 @@
-	$('.yourTurn').text('it is your Turn!');
-	$('.startGameRow').replaceWith('<h3 class="ftw">For the Win!</h3>');
-	$('.answer').after('<button class="submitAns" disabled="true">ROCK ON</button>')
-	$('.showAns').attr('disabled', false);
-	$('.submitAns').attr('disabled', false);
-	$('.answer').attr('disabled', false);
-	$('.reference li:first-child').replaceWith('<li class="press"><div class="pressA" >pacifica</div> <p><span>enter</span>A<span>/a</span></p> </li>');
-	$('.reference li:nth-child(2)').replaceWith('<li class="press"><div class="pressB">caribic</div> <p><span>enter</span>B<span>/b</span></p> </li>');
-	$('.reference li:nth-child(3)').replaceWith('<li class="press"><div class="pressC">Chic</div> <p><span>enter</span>C<span>/c</span></p> </li>');
-	$('.reference li:nth-child(4)').replaceWith('<li class="press"><div class="pressD">Cheery</div> <p><span>enter</span>D<span>/d</span></p> </li>');
-	$('.reference li:nth-child(5)').replaceWith('<li class="press"><div class="pressE">Babyblue</div> <p><span>enter</span>E<span>/e</span></p> </li>');
-};
-
-var displayPlayer = function (){
-	if ($('tr.answerList').length%2 === 0) {
-		$('.whoseTurn').text(playerTwoName + ',');
-		currentPlayer = playerTwoName;
-	} else {
-		$('.whoseTurn').text(playerOneName + ',');
-		currentPlayer = playerOneName;
-	}
-	return currentPlayer;
-};
-
-var genTurnCount = function (){
-	turnCount = $('.answerList').length;
-	var currentTurn = $('.answerList:last-child td .turnNum');
-	$(currentTurn).text(turnCount);
-};
-
-// Get the solution
-var randomColor = function () {
-	var solu = color[Math.floor(Math.random()*color.length)];
-	return solu;
-};
-
-var generateSolution = function (){
-	for (var i = 0; i < 4; i++) {
-		solutionArray[i] = randomColor();
-	}
-	return solutionArray;
-};
-
-var drawSolution = function (){
-	var drawSolutionArray = [];
-	for (var x = 0; x < solutionArray.length; x++) {
-		var eachSolution = solutionArray[x];
-		switch(eachSolution) {
-			case 'pacifica':
-				drawSolutionArray.push('<div class="solutionBullet pacificaColor" style="display: none"></span>');
-				break;
-			case 'caribic':
-				drawSolutionArray.push('<div class="solutionBullet caribicColor" style="display: none"></span>');
+	//////////////////////////////
```

Comment:
as a general rule, please use [2 spaces](http://www.sublimetext.com/docs/indentation) for indentation instead of 4